apps/site/assets/css/_banner.scss

```diff
@@ -171,3 +171,22 @@
 .c-banner__date {
   font-weight: bold;
 }
+
+.c-focus--bus {
+  &:focus {
+    color: inherit;
```

Comment:
Not 100% sure about this but it might make sense to explicitly use the `$black` color here.

Reply:
I think for buses it's a specific shade of dark blue, that is why Adamo suggested in the ticket to use `inherit`.
@amaisano can you please confirm? otherwise `$black` totally makes sense to me :)

Reply:
I don't recall if I meant "inherit" in the literal property sense. I do remember thinking the color combos (white on red, white on blue, etc) were already present in the CSS hierarchy, so that all we really needed to do was remove hard-coded text color properties, and/or use the `:hover, :focus { color: inherit; }` rule so that the pseudo state text color matched the static state text color, without needing to hard code every mode (which I do not like).

Reply:
PS: I'm sorry this ticket was so vague. I'm sure I knew exactly how to fix it when I wrote it 😬